src/WheelZoomExtension.cpp

```diff
@@ -0,0 +1,36 @@
+#include "WheelZoomExtension.h"
+
+#include <QDateTime>
+
+static const int SCROLL_TIMEOUT = 1000; // in milliseconds
+
+WheelZoomExtension::WheelZoomExtension(TextEdit *textEdit)
+    : TextEditExtension(textEdit), mPartialDelta(0), mLastUpdate(0)
```

Comment:
I prefer initializing scalar members in the class declaration, this way if new constructors are added the initial values do not have to be duplicated (admittedly, it's unlikely that this class will get a new constructor, but still)

Reply:
Thanks! My C++ is a little rusty and mostly pre-C++11, so I do really appreciate your helpful comments! :)